content/installation/administrator/admin_install_git/index.md

```diff
@@ -25,15 +25,16 @@ De installatiebestanden zijn beschikbaar via http://git-scm.com/downloads
 1. Gebruik de standaard door _Next_ te klikken.
 1. Gebruik de standaard door _Install_ te klikken.
 1. Vink alles uit en klik op _Next_.
-1. Download [.gitconfig](.gitconfig).
+1. Download [gitconfig](gitconfig).
 Open het bestand met een editor en pas de `name` en `email` aan.
-Bewaar het bestand in `c:/users/username`.
+Bewaar het bestand als `.gitconfig` (bestandsnaam start met een punt) in `c:/users/username`.
```

Comment:
I notice the `.gitconfig` file also stores information such as the fact that the ssh connection should be shown (or used?) on Github instead of https:://github.com/.  This one is not problematic to lose, I think (unless it also triggers to use ssh where it is set up), but it may be useful to check if more this file can also contain more important information.  (Same for `Rprofile.site`)

Reply:
> I'm confused. `gitconfig` doesn't contain any reference to ssh.

In my case this file contains a reference to ssh:
```
[url "ssh://git@github.com/"]
        insteadOf = https://github.com/
[user]
        email = els.lommelen@inbo.be
        name = ElsLommelen
```
I don't know if this is important for anything but the link that is shown on Github?  Anyhow, if removing this has important consequences, I rather don't want to have this file replaced.

And also: my login name on Github is without spaces, should this be the same here?  A lot of INBO Github users have a nickname different from `Voornaam Achternaam`, so if it is important that this exact name is stored in the gitconfig file (to be able to push changes or other functionality), it seems a bad idea to overwrite `.gitconfig` with a default.

So, in general: if changing the present version of the file can have any (important) consequences on the functionality for the user, I would be careful to just replace it by a default that doesn't contain this specific part.

Reply:
Maybe we should move the `.gitconfig` stuff to the user part of the installation. So the user can append or update the relevant parts.

BTW  `user.name` can be any name you like. You don't have to (but can) use your github user name.

Reply:
This is certainly a nice option for users that like to customize their installation (or that for some reason have an aberrant file).  Drawback is that it is an extra effort for everyone, and that it might need some explanation.  (I could on the other hand refer to it in the github workflow, so users can do this at the moment they understand why it is useful.)

Good to know that this user name doesn't have to be the same as the github user name, this reduces the need for a custom approach...

Reply:
As a first time user you'll need to specify name and password. The easiest option it to set this `.gitconfig`. It's a good idea to refer to it in the github workflows.